src/main/java/appeng/integration/abstraction/IJEI.java

```diff
@@ -1,15 +1,25 @@
 package appeng.integration.abstraction;
 
 
+import appeng.integration.IIntegrationModule;
+
+
 /**
  * Abstracts access to the JEI API functionality.
  */
-public interface IJEI
+public interface IJEI extends IIntegrationModule
 {
 
-	boolean isEnabled();
+	default String getSearchText()
+	{
+		return "";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this method and declare a constant for this value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS3400)